### PR TITLE
Add documentation for the xwiki official image

### PR DESCRIPTION
**Pull request by vmassol**

### CODE REVIEW

**Comment by vmassol** [comment]
@yosifkit Thanks for the review and the commits. I've just updated the doc too to align with the latest version from https://github.com/xwiki-contrib/docker-xwiki/blob/master/README.md

**Comment by vmassol** [comment]
Apparently I made a mistake since the build is failing now :(

**Comment by vmassol** [comment]
@yosifkit sorry I broke the build. I tried to fix it but I don't see the mistake I made. I guess I'll need your help once more... sorry

**Comment by yosifkit** [comment]
No problem. The markdownfmt is very particular, but ensures specific formatting. :smiley_cat:  It looks like in this case it was needing an empty line between a paragraph and the list after it, needing each paragraph to take only one line, and needing to remove a trailing space on a line.

**Comment by vmassol** [comment]
ok! Thanks for fixing it :)

File: xwiki/content.md [hunk 1]
```diff
@@ -57,7 +55,7 @@ services:
       - MYSQL_USER=xwiki
       - MYSQL_PASSWORD=xwiki
   db:
-    image: "mysql:5"
+    image: "mysql:5.7"
```
Comment (vmassol) [inline xwiki/content.md]: So `latest` needs to have all dependencies with a fixed versoin and it's only the main image software that doesn't have its version fixed?

File: xwiki/content.md [hunk 2]
```diff
@@ -17,8 +17,6 @@ The goal is to provide a production-ready XWiki system running in Docker. This i
 
 # How to use this image
 
-[![](https://images.microbadger.com/badges/version/xwiki/xwiki-mysql-tomcat.svg)](https://microbadger.com/images/xwiki/xwiki-mysql-tomcat) [![](https://images.microbadger.com/badges/image/xwiki/xwiki-mysql-tomcat.svg)](https://microbadger.com/images/xwiki/xwiki-mysql-tomcat)
-
```
Comment (vmassol) [inline xwiki/content.md]: ok I'll need to remember to remove those when in the future I'll sync the README from https://github.com/xwiki-contrib/docker-xwiki with the official image README! :)

**Pull request merged**